### ISSUE TITLE
Rephrase fallbacks

### ISSUE THREAD

**Issue by jvehent** [jvehent/cipherscan#87]
I'd like to rephrase the output of the tls intolerance test. the current output isn't very clear on what the information means:

```
Fallbacks required:
big-SSLv3 config not supported, connection failed
big-TLSv1.0 no fallback req, connected: TLSv1 AES128-SHA
big-TLSv1.1 no fallback req, connected: TLSv1 AES128-SHA
big-TLSv1.2 no fallback req, connected: TLSv1 AES128-SHA
```

* `connection failed` make it sound as if something bad happened, when in fact not having sslv3 is a good thing
* `no fallback req, connected` doesn't really say if the connection used what was expected, or if any intolerancy was detected

Maybe we could compress the terminal output by just showing `Intolerancies: none [OK]` when everything goes well, and dive into details when it doesn't.

@tomato42 : Thoughts?

**Comment by tomato42** [comment]
Then we would need to detect when a server negotiates TLS1.1 when given TLS1.2 Client Hello, but negotiate TLS1.0 when given TLS1.0.

Or one that negotiates SSLv3 when provided with any TLS Client Hello version except TLS 1.0. And with TLS 1.0 it negotiates TLS 1.0.

This would require *a lot* of logic, I'd suggest leaving the detailed report in.

For rephrasing: we could say `connection unsuccessful` instead, but that doesn't change much, user still needs to connect the two pieces of information together - SSLv3 and "failed".

for `no fallback req, connected`, yes, it doesn't, but analysing it is not simple, as I explained above.

**Comment by jvehent** [comment]
Looking through the code, I can't find a straighforward way to implement this either. I like having the data, but it bothers me that the output will mean nothing to most people who use the tool. Clarity is important, as the goal is to provide sysadmins with a quick overview of their config.
TLS Intolerances are an edge case that really shouldn't happen on a normal setup, so for 99% of our users, this test will never return anything meaningful (hopefully). I propose that we disable it by default and only run it when asked, using a `--test-intolerances` flag.
We could also enable it by default when `-j` is used.

**Comment by tomato42** [comment]
In analysing the results, I'm mostly looking at the result for TLS1.2 Client Hello - if I get any error there, or it doesn't give TLSv1.2 protocol that rises a red flag.

So if we want to make the output clearer, we can do a simple check for `big-TLSv1.2` == `TLS1.2` and print the results if it's not the case. That would mean running the tests always.

**Comment by jvehent** [comment]
Implemented in #89

**Comment by tomato42** [comment]
if you put in the opening comment or in a comment of any of the patches a phrase "fixes #89" github will close the bug automatically once the PR is merged :)
